golang/cosmos/cmd/libdaemon/main.go

```diff
@@ -92,6 +121,8 @@ func RunAgCosmosDaemon(nodePort C.int, toNode C.sendFunc, cosmosArgs []*C.char)
 	// fmt.Fprintln(os.Stderr, "Starting Cosmos", args)
 	os.Args = args
 	go func() {
+		defer handlePanic("daemon.RunWithController")
```

Comment:
Why do we need this `handlePanic`? Isn't it a goroutine in the first place?

Reply:
My intent was to guarantee the exit of the program if this goroutine panics, but I may have been overly defensive.

So, it may not be strictly necessary, but shouldn't hurt.